fern/main.css

```
@@ -0,0 +1,316 @@
+.fern-footer-nav {
+  border-radius: var(--rounded);
+  background-color: transparent !important;
+  transition:
+    box-shadow 0.3s ease,
+    outline 0.3s ease;
+}
+
+.fig-caption {
+  --callout-border: transparent !important;
+  --callout-icon-color: var(--grayscale-a8) !important;
+  background-color: var(--nv-light-grey-1) !important;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)